baseplate/experiments/__init__.py

```diff
@@ -23,26 +22,33 @@ class ExperimentsContextFactory(ContextFactory):
     This factory will attach a new :py:class:`baseplate.experiments.Experiments`
     to an attribute on the :term:`context object`.
     """
-    def __init__(self, path, event_queue):
+    def __init__(self, path, event_logger=None):
         """ExperimentContextFactory constructor
 
         :param str path: Path to the experiment config file.
         :param baseplate.events.EventQueue event_queue: Event queue used for
             logging bucketing events to the event pipeline.  You can set this
-            to None to disable bucketing events entirely.
+            to None to disable bucketing events entirely. This queue should be
+            configured to use baseplate.events.serialize_v2_event
         """
         self._filewatcher = FileWatcher(path, json.load)
-        self._event_queue = event_queue
+        self._event_logger = event_logger
 
-    def make_object_for_context(self, name, server_span):
+    def make_object_for_context(self, name, server_span, event_logger=None):
         return Experiments(
             config_watcher=self._filewatcher,
-            event_queue=self._event_queue,
             server_span=server_span,
             context_name=name,
+            event_logger=self._event_logger
         )
 
 
+class V2DebugLogger(EventLogger):
```

Comment:
I would just call this `DebugEventLogger` since it doesn't really care about v2 events.

Reply:
Also, I think it makes sense to put this in `events.py` rather than here since it's not specific to experiments.

Reply:
fixed